### PR TITLE
Add support for Stork search

### PR DESCRIPTION
**Pull request by alexander-falca**
## Description

This PR adds support for [Stork](https://stork-search.net/) search.  Updated example site to use Stork.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: exampleSite/config.yaml [hunk 1]
```diff
@@ -18,10 +18,15 @@ params:
   fuseSearch: true
```
Comment (amuraru) [inline exampleSite/config.yaml]: do we need this switch anymore?

File: assets/js/stork-search.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ *
+ * NOTICE: Adobe permits you to use, modify, and distribute this file in
+ * accordance with the terms of the Adobe license agreement accompanying
+ * it. If you have received this file from a source other than Adobe,
+ * then your use, modification, or distribution of it requires the prior
+ * written permission of Adobe.
+ *
+ */
+const initPromise = stork.initialize("{{ .Site.BaseURL }}/stork/stork.wasm");
+const downloadPromise = stork.downloadIndex("site-search-index", "{{ .Site.BaseURL }}/stork/site-index.st");
```
Comment (amuraru) [inline assets/js/stork-search.js]: should we add `{ forceOverwrite: true }` ?

**Review by amuraru — APPROVED** [review]
nice one! thanks Alex

File: exampleSite/config.yaml (hunk 1 above)
Comment (alexander-falca) [reply exampleSite/config.yaml]: no, we don't need. removed.

File: assets/js/stork-search.js (hunk 2 above)
Comment (alexander-falca) [reply assets/js/stork-search.js]: yep, good idea. added.

**Review by alexander-falca — COMMENTED** [review]
thanks for review @amuraru!

**Pull request merged**